File: ruby/getting-localized-content/GetLocalizedItemsWithSlug.rb
# DocSection: getting_localized_content_url_slug
# Tip: Find more about Ruby SDKs at https://docs.kontent.ai/ruby
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '8d20758c-d74c-4f59-ae04-ee928c0816b'
delivery_client.items([
                        'system.type'.eq('article'),
                        'elements.url_pattern'.eq('acerda-de-nosotros')
                      ])
               .language('es-ES')
               .execute do |response|
                 items = response.items
               end
# EndDocSection

File: js/getting-localized-content/GetLocalizedItemsWithSlug.js
// DocSection: getting_localized_content_url_slug
// Tip: Find more about JS/TS SDKs at https://docs.kontent.ai/javascript
const KontentDelivery = require('@kentico/kontent-delivery');

const deliveryClient = new KontentDelivery.DeliveryClient({
  projectId: '8d20758c-d74c-4f59-ae04-ee928c0816b7',
  typeResolvers: [
    // Create strongly typed models according to https://docs.kontent.ai/strongly-typed-models
    new KontentDelivery.TypeResolver('article', (rawData) => new Article)
  ]
});

deliveryClient.items('article')
  .type('article')
  .languageParameter('es-ES')
  .depthParameter(0)
  .equalsFilter('elements.url_pattern', 'acerda-de-nosotros')
  .toObservable()
  .subscribe(response => console.log(response));
// EndDocSection